engine-nextgen/src/main/kotlin/graphql/nadel/NextgenEngine.kt

```diff
@@ -97,27 +100,63 @@ class NextgenEngine @JvmOverloads constructor(
         }
 
         val executionContext = NadelExecutionContext(executionInput, serviceExecutionHooks)
+        val instrumentationContext = getInstrumentationContext(query, queryDocument, executionInput, instrumentationState)
```

Comment:
this is ported from here https://github.com/atlassian-labs/nadel/blob/master/engine/src/main/java/graphql/nadel/engine/execution/Execution.java#L105